### PR TITLE
Deployment update + repo cleanup

### PR DESCRIPTION
**Pull request by Dzejkop**
Adds:
1. A docker workflow that produces a ready to use docker image with all the contracts deployed
2. `forge script` based deployment scripts
3. Vendors the verifier contracts for all batch sizes
4. Adds a release workflow (on tag)

### CODE REVIEW

File: script/Deploy.s.sol [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+
+    address batchInsertionVerifiers = address(0);
+    address batchDeletionVerifiers = address(0);
+
+    function run() external returns (address router, address worldIDOrb, address worldIDPhone) {
+        console.log("Deploying WorldIDRouter, WorldIDOrb, and WorldIDPhone");
+
+        WorldIDIdentityManager worldIDOrb = deployWorldID(INITIAL_ROOT);
+        console.log("WorldIDOrb:", address(worldIDOrb));
+        WorldIDIdentityManager worldIDPhone = deployWorldID(INITIAL_ROOT);
```
Comment (0xKitsune) [inline script/Deploy.s.sol:35]: Do we need to deploy the phone credential if the phone sequencer is offchain?

File: script/Deploy.s.sol [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+        console.log("WorldIDPhone:", address(worldIDPhone));
+
+        WorldIDRouter router = deployWorldIDRouter(IWorldID(address(worldIDPhone)));
```
Comment (0xKitsune) [inline script/Deploy.s.sol:38]: Consider logging the `WorldIDRouter` address after deployment.

**Review by 0xKitsune — APPROVED** [review]

File: script/Deploy.s.sol (hunk 1 above)
Comment (Dzejkop) [reply script/Deploy.s.sol:35]: Good point, we don't!

**Review by thechillday1 — COMMENTED** [review]
Hmm

**Pull request merged**